lib/fathead/caniuse/redirects.txt

```diff
@@ -0,0 +1,976 @@
+svg filters,svg-filters
+svg filters,svg-filters
+svg filters,svg-filters
```

Comment:
@anjanjr looks like there are 3 of every redirect in this file?

Please fix. 

Alternatively, seeing as you're generating the names of the redirects in the parser, it may be simpler to just create redirect entries in `output.txt` rather than generate this file